lib/pact_broker/client/cli/broker.rb

```diff
@@ -122,7 +122,7 @@ def describe_version
         method_option :contract_content_changed, type: :boolean, desc: "Trigger this webhook when the pact content changes"
         method_option :provider_verification_published, type: :boolean, desc: "Trigger this webhook when a provider verification result is published"
         method_option :verbose, aliases: "-v", type: :boolean, default: false, required: false, desc: "Verbose output. Default: false"
-        method_option :uuid, type: :numeric, required: false, desc: "A uuid for the webhook"
+        method_option :uuid, type: :string, required: false, desc: "Optionally specify a uuid for the webhook"
```

Comment:
Let's make this more helpful and let the user know what the implication is of providing the UUID (ie. this becomes an upsert).

Reply:
Actually, strike that. Let's make a new command - see my comment here: https://github.com/pact-foundation/pact_broker-client/issues/54#issuecomment-577524524